setup.cfg

```diff
@@ -28,6 +28,8 @@ install_requires =
     importlib-metadata;python_version < "3.8"
     typing;python_version<"3"
 python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*
+package_dir =
```

Comment:
I usually use `packages = find:` and then add another table:

```ini
[options.packages.find]
where = src
```

In this case it doesn't matter, but *not* doing that is one of the leading causes of the problems that get caught when you test in tree rather than testing the installed package. :wink:

We can leave that for later, though.

Reply:
Sure, I can add that. It would need to be tested by installing the package either way, `find` has some non-obvious behaviors that we need to keep in mind, for eg. I am pretty sure last time I tried it didn't install submodules if the "super"module did not have an `__init__.py`.